extensions/patterns/configure_ec2/exec_env/execution-environment.yml

```diff
@@ -0,0 +1,29 @@
+---
+version: 3
+
+dependencies:
+  ansible_core:
+    package_pip: ansible-core
+  ansible_runner:
+    package_pip: ansible-runner
+  galaxy:
+    collections:
+      - name: cloud.aws_ops
+        source: https://github.com/redhat-cop/cloud.aws_ops.git
+        type: git
+        version: main
+      - name: amazon.aws
+        source: https://github.com/ansible-collections/amazon.aws.git
+        type: git
+        version: main
```

Comment:
Yeah I can def do that for amazon.aws. For cloud.aws_ops do we want to leave as is in case there are unreleased changes to the pattern that we want available?

Reply:
I agree in general, although I'd suggest maybe we make a ticket to update that and do a release after all the MVP work is finished?

Reply:
Ticket created https://issues.redhat.com/browse/ACA-2042.